.github/ISSUE_TEMPLATE/06-training.yml

```diff
@@ -61,39 +73,13 @@ body:
         - ROCm 4.3.1
         - oneDNN

```

Comment:
remove oneDNN, and just keep "CUDA" and "ROCm" - versions can be specified in the EP version field